extensions/mqtt/processors/AbstractMQTTProcessor.h

```diff
@@ -28,16 +28,16 @@
 #include "core/logging/LoggerConfiguration.h"
 #include "MQTTClient.h"
 
-#define MQTT_QOS_0 "0"
-#define MQTT_QOS_1 "1"
-#define MQTT_QOS_2 "2"
-
 namespace org {
 namespace apache {
 namespace nifi {
 namespace minifi {
 namespace processors {
 
+#define MQTT_QOS_0 "0"
```

Comment:
I agree that these would be better at a namespace/class scope, but defines are not influenced by namespaces, so this change has no effect. Perhaps a constexpr?

Reply:
Just wanted to put the definitions together, but constexpr is a fair point.
Changed them.